pkg/controller/controller_binding.go

```diff
@@ -582,8 +615,8 @@ func (c *controller) reconcileServiceInstanceCredential(binding *v1alpha1.Servic
 			}
 			s := fmt.Sprintf(
 				"Error unbinding ServiceInstanceCredential \"%s/%s\" for ServiceInstance \"%s/%s\" of ServiceClass %q at ServiceBroker %q: %s",
-				binding.Name,
 				binding.Namespace,
+				binding.Name,
 				instance.Namespace,
 				instance.Name,
 				serviceClass.Spec.ExternalName,
```

Comment:
I think there needs to be logic that ensures that the ready condition communicates to the user when orphan mitigation has failed.